modules/core/jvm/src/main/scala/com/azavea/stac4s/SummaryValue.scala

```diff
@@ -0,0 +1,66 @@
+package com.azavea.stac4s
+
+import cats.kernel.Eq
+import cats.syntax.all._
+import eu.timepit.refined.types.string
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.refined._
+import io.circe.schema.Schema
+import io.circe.{Decoder, DecodingFailure, Encoder, HCursor, Json}
+
+import scala.util.Try
+
+sealed abstract class SummaryValue
+
+case class StringRangeSummary(
+    minimum: string.NonEmptyString,
+    maximum: string.NonEmptyString
+) extends SummaryValue
+
+case class NumericRangeSummary(
+    minimum: Double,
+    maximum: Double
+) extends SummaryValue
+
+case class SchemaSummary private[stac4s] (
+    underlying: Json
+) extends SummaryValue {
+  val schema: Schema = Schema.load(underlying)
```

Comment:
Where is it used? Do we need it here?

Reply:
Uhhh I guess not, I was going to call it in the decoder but I'm calling load directly. Will remove